appveyor.yml

```diff
@@ -60,15 +60,11 @@ install:
 
   # Install the dependencies of the project.
   - ps: Add-AppveyorMessage "Installing conda packages..."
-  - "%CMD_IN_ENV% conda install -yq %TEST_DEPS%"
-  - "%CMD_IN_ENV% conda install -yq %DEPS%"
-  - "pip install pytest-mpl"
-
-  # Having 'sip' folder on path confuses import of `sip`.
-  #- "%CMD_IN_ENV% conda install -yq pip"
+  - "conda install -yq -c conda-forge %TEST_DEPS%"
```

Comment:
Why should we use conda-forge to install all the deps? In a standard installation most people will have downloaded the full anaconda (vs miniconda here), hence most of the dependencies would be already installed and coming from the standard anaconda repos.

Reply:
This was part of removing the "test" optional dependency installation, so this should be removed now but pytest-cov still need to be installed.

Reply:
My point was more about why conda-forge instead of the standard anaconda repository.

Reply:
pytest-mpl is not in defaults.

Reply:
OK